### PR TITLE
Update template to SDK 0.6.1

### LINKED ISSUE #4: Repo outdated
It has come to my attention that this version of the repo is outdated.

Can we explicitly mark it as such and point to the right repo? Or delete this one?

### PR DESCRIPTION
**Pull request by remco-nfi**
Resolves #4 

* This updates the template to SDK version 0.6.1
* It also (finally) adds a Dockerfile to the template
* setup.py has been removed, dependency management is now recommended to be done in requirements.in

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,32 @@
 # Hansken Extraction Plugin skeleton for Python based on version 0.4.11
```
Comment (blackbeard334) [inline README.md]: don't forget to update this version as well 🙃

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Multi-stage Dockerfile, to build and package an extraction plugin
```
Comment (blackbeard334) [inline Dockerfile:1]: +1 love these comments

**Review by blackbeard334 — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (remco-nfi) [reply README.md]: Thx, done!

**Review by GijsPannebakker — APPROVED** [review]

**Review by blackbeard334 — APPROVED** [review]

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+#  Recommended way to build the plugin is by calling tox:
+#    tox -e package
+#  if you need to pass a proxy:
+#    tox -e package -- --build-arg https_proxy=https://your-proxy
+#  if you want to a private package index:
```
Comment (GijsPannebakker) [inline Dockerfile]: typo: if you want to a?

Comment (bartbroere) [reply Dockerfile]:
```suggestion
#  if you want to pass a private Python package index:
```
Something like this?

File: run_with_hanskenpy.py [hunk 4]
```diff
@@ -1,6 +1,6 @@
 from hansken_extraction_plugin.runtime.extraction_plugin_runner import run_with_hanskenpy
 
-from plugin.extraction_plugin import Plugin
+from plugin.plugin import Plugin
```
Comment (bartbroere) [inline run_with_hanskenpy.py]: This perhaps might be one time too often the word `plugin`. Could this perhaps be resolved by adding an `__init__` or `__all__` somewhere?

File: run_with_hanskenpy.py [hunk 5]
```diff
@@ -1,6 +1,6 @@
...
 
 if __name__ == '__main__':
```
Comment (bartbroere) [inline run_with_hanskenpy.py]: I like to simply put the `run_with_hanskenpy` call in the plugin itself within the `if __name__ == '__main__'` check. The `serve_plugin` doesn't seem to mind.

Comment (bartbroere) [reply run_with_hanskenpy.py]: https://github.com/NetherlandsForensicInstitute/audio-segmentation/blob/master/audio_segmentation.py#L197-L198

File: Dockerfile (hunk 2 above)
Comment (bartbroere) [inline Dockerfile:1]: Nice to see the `Dockerfile` reintroduced in the template. This gives plugin developers a lot more flexibility. It allows different base images, extra libraries etc.

**Review by bartbroere — APPROVED** [review]

File: run_with_hanskenpy.py (hunk 4 above)
Comment (remco-nfi) [reply run_with_hanskenpy.py]: I'll move the main method to `plugin.py` as you suggested

File: run_with_hanskenpy.py (hunk 5 above)
Comment (remco-nfi) [reply run_with_hanskenpy.py]: will do!

File: Dockerfile (hunk 3 above)
Comment (remco-nfi) [reply Dockerfile]: thanks

File: Dockerfile (hunk 2 above)
Comment (remco-nfi) [reply Dockerfile:1]: The template never had a `Dockerfile`, so it is the first introduction. :)
It was never intended to not have a `Dockerfile` when writing a plugin.

Comment (bartbroere) [reply Dockerfile:1]: It was there in the very first commit it seems :-). But that might have been very short-lived indeed.

**Pull request merged**